base/dotenv.default.mustache

```diff
@@ -0,0 +1,3 @@
+PORT=
+RETHINK_HOST=
+RETHINK_NAME=
```

Comment:
.env should be in .gitingore so it won't be committed?  So I think creating a .env would be okay?

Hmm I'm still concerned with the naming of "example".. just sounds like turfable file/code.